Settings.as

```diff
@@ -1,12 +1,14 @@
+#if SIG_DEVELOPER
+
```

Comment:
Don't guard the settings with `SIG_DEVELOPER` - this will clear user settings outside of developer mode.